lib/templates/updateDevice.json

```diff
@@ -177,6 +177,10 @@
         }
       }
     },
+    "internal_attributes": {
+      "description": "Free form array of internal attributes for particular IOTAgents",
+      "type": "any"
```

Comment:
Why any instead of array type?

Reply:
To be aligned with the choice made in https://github.com/telefonicaid/iotagent-node-lib/blob/master/lib/templates/createDevice.json, line 197 (we must allow the same structure as the structure allowed when creating the device).